### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by soulitzer**
Update readme's old example and also include a new example featuring the latest update.

Preview readme changes at https://github.com/soulitzer/pytorchviz/tree/update-docs
See the update to jupyter notebook here: https://github.com/szagoruyko/pytorchviz/blob/71111ea7687ea6ceb0b74d3dd310abf6eb308111/examples.ipynb

The jupyter notebook update is turning out to be a little tricky. So far my workaround is that I'm not running all the cells (leaving some cells out of date) because 1) the jit examples have been failing since forever, and also 2) for larger graphs I'm finding it hard to replicate the scaling from the original notebook (which is able to fit the entire graph in the view), even with code from older commits. Haven't ruled out factors such as different notebook versions, graphviz version or maybe even screen resolution.

cc @szagoruyko @albanD

### CODE REVIEW

**Review by albanD — APPROVED** [review]
The update looks good to me. Even though checking the diff on the notebook is hard so I may have missed something.

@szagoruyko this is a bit more verbose but we think these will be useful examples.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,33 @@ pip install torchviz
...
+
+x = torch.randn(1, 8)
+y = model(x)
+
+make_dot(y.mean(), params=dict(model.named_parameters()))
+```
+![image](https://user-images.githubusercontent.com/13428986/110844921-ff3f7500-8277-11eb-912e-3ba03623fdf5.png)
 
-<img width="891" alt="screen shot 2018-01-30 at 16 13 01" src="https://user-images.githubusercontent.com/4953728/35574234-8780297e-05d9-11e8-8e80-f4009297cefd.png">
+Set `show_attrs=True` and `show_saved=True` to see what autograd saves for the backward pass.
```
Comment (albanD) [inline README.md:36]: We should add a note that this is only available for pytorch 1.9+ here as well.

Comment (soulitzer) [reply README.md]: Added!

**Review by albanD — APPROVED** [review]
LGTM ready to merge

**Comment by szagoruyko** [comment]
thanks so much @soulitzer @albanD !

**Pull request merged**